pydocx/export/base.py

```diff
@@ -779,7 +826,7 @@ def paragraph(self, text):
         return text
 
     @abstractmethod
-    def heading(self, text, heading_level):
+    def heading(self, text, heading_style_name):
```

Comment:
I think so. We don't need to bump the major version, since we're not 1.0.0, but it's pretty easy to communicate something like this and would save our early users (usually the most important users) a lot of potential headache.